tests/test_client.py

```diff
@@ -22,36 +22,6 @@ def test_push_job(self):
         self.assertEqual(m.get('state'), u'deleted')
         self.assertEqual(m.get('foo'), u'baz')
 
-    def test_botgroup(self):
```

Comment:
In keeping case we can save the other one as well - ``test_start_job``. Only fix assertion ``        self.assertEqual(dict(j1.metadata), dict(j2.metadata))``, because it would fail as ``start_job`` returns auth key and ``get_job`` not. So with something like ``j1.metadata.pop('auth')``.